### ISSUE TITLE
Decide what to do with intro_vp vignette

### ISSUE THREAD

**Issue by peterdesmet** [adokter/bioRad#164]
There was an `intro_vp.Rmd` vignette written by @plieper and @stijnvanhoey in 2017. Because it eventually contained outdated function names, its [last version](https://github.com/adokter/bioRad/blob/42ee484ce491fdbc94080991061682fcee875407/vignettes/intro_ppi.Rmd) was removed in 8f638d59b9148b6b6e37ae781088b1b1d4fd1926.

It should be checked if the functionality it described is covered elsewhere (e.g. in function examples: [download_vpfiles.R](https://github.com/adokter/bioRad/blob/master/R/download_vpfiles.R)). If not, it should be resurrected 🧟‍♂️ as a vignette, probably with the name `download_vpfiles.Rmd`.

**Comment by adokter** [comment]
This vignette has been superseded by https://github.com/adokter/bioRad/blob/master/vignettes/RadAero18.Rmd, and I think it does not need to be resurrected.

The ENRAM specific functionality to interact with online repository of vertical profiles (e.g. [download_vpfiles.R](https://github.com/adokter/bioRad/blob/master/R/download_vpfiles.R))) is however not included in any of the examples at the moment. We might want to make a specific vignette for that.

**Comment by peterdesmet** [comment]
Agreed, let’s leave this issue open for that. Might change with GloBAM + would be good to include how to download from NEXRAD as well. Will update title.

**Comment by adokter** [comment]
Regarding NEXRAD, see issue #41

**Comment by adokter** [comment]
stale, should be tackled in getRad
